app/views/locations/new.html.erb

```diff
@@ -0,0 +1,63 @@
+<% content_for :title, "Add a location to this scheme" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+        text: "Back",
+        href: "/schemes/#{@scheme.id}/support",
+      ) %>
+<% end %>
+
+<%= render partial: "organisations/headings", locals: { main: "Add a location to this scheme", sub: @scheme.service_name } %>
+
+<%= form_for(@location, method: :post, url: location_create_scheme_path) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_text_field :postcode,
+                             label: { size: "m" },
+                             hint: { text: "For example, SW1P 4DF." },
+                             width: 5 %>
+
+      <%= f.govuk_text_field :name,
+                             label: { text: "Name (optional)", size: "m" },
+                             hint: { text: "This is how you refer to this location within your organisation" } %>
+
+      <%= f.govuk_number_field :total_units,
+                               label: { text: "Total number of units at this location", size: "m" },
+                               width: 2,
+                               hint: { text: "A unit can be a bedroom in a shared house or flat, or a house with 4 bedrooms. Do not include bedrooms used for wardens, managers, volunteers or sleep-in staff.s" },
+                               autofocus: true %>
+
+      <% type_of_units_selection = Location.type_of_units.keys.map { |key, _| OpenStruct.new(id: key, name: key.to_s.humanize) } %>
+
+      <%= f.govuk_collection_radio_buttons :type_of_unit,
+                                           type_of_units_selection,
+                                           :id,
+                                           :name,
+                                           legend: { text: "What is this type of scheme?", size: "m" } %>
+
+      <% wheelchair_user_selection = Location.wheelchair_adaptations.keys.reverse.map { |key, _| OpenStruct.new(id: key, name: key.to_s.humanize) } %>
```

Comment:
Worth just defining these the right way around in the model since we call `.reverse` everywhere we use it?

Reply:
Done